App/iOS/Delegates/AppDelegate.swift

```diff
@@ -469,6 +474,34 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   func sceneInfo(for sceneSession: UISceneSession) -> SceneInfoModel? {
     return sceneInfo
   }
+  
+  private nonisolated func cleanUpWebKitPendingFileUploads() async {
+    // When a user uploads a file through a web page, WebKit will copy said file to the `/tmp` directory in
+    // the app sandbox. If the user does not close the tab they used to upload (thus destroying the WKWebView)
+    // before the app terminates, then WebKit no longer cleans up the file and it remains on disk until
+    // the OS decides to purge /tmp directories to recover disk space. This function will attempt to remove
+    // known temporary folders that WebKit creates to clean up any files that may be lingering there due to
+    // this WebKit bug.
+    let fileManager = FileManager.default
+    let tmp = fileManager.temporaryDirectory
+    let knownFolders: Set<String> = ["WKWebFileUpload", "WKFileUploadPanel", "WKVideoUpload"]
```

Comment:
@kylehickinson Are these all in the WKFileUploadPanel form?

Reply:
I know for sure that `WKFileUploadPanel` is created in root folder, but based on user reports I am guessing the other 2 are also created in some form as well. Perhaps from older iOS versions? Or specific types of file pickers on websites?

Reply:
This method at least describes at least `WKFileUploadPanel` and `WKVideoUpload` https://github.com/WebKit/WebKit/blob/0c6de669a59bddb53aa318ddf458c700336c6141/Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.mm#L354

Reply:
Sounds perfect for this particular use-case.

Reply:
And this one `WKFileUploadPanel`: https://github.com/WebKit/WebKit/blob/0c6de669a59bddb53aa318ddf458c700336c6141/Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.mm#L1144